build/readme-parser.ts

```diff
@@ -42,12 +42,12 @@ function extractDataFromMatch(match: RegExpMatchArray): FeatureMeta {
 
 export function getFeaturesMeta(): FeatureMeta[] {
 	const readmeContent = readFileSync('readme.md', 'utf-8');
-	return [...readmeContent.matchAll(featureRegex)].map(match => extractDataFromMatch(match));
+	return [...readmeContent.matchAll(featureRegex)]
+		.map(match => extractDataFromMatch(match))
+		.sort((firstFeature, secondFeature) => firstFeature.id.localeCompare(secondFeature.id));
 }
 
-export function getFeatures(): FeatureID[] {
+export function getImportedFeatures(): FeatureID[] {
 	const contents = readFileSync('source/refined-github.ts', 'utf-8');
-	return [...contents.matchAll(/^import '\.\/features\/([^.]+)';/gm)]
-		.map(match => match[1] as FeatureID)
-		.sort();
+	return [...contents.matchAll(/^import '\.\/features\/([^.]+)';/gm)].map(match => match[1] as FeatureID);
```

Comment:
The bugs were caused by the decoupling of `getFeatures()` and `getFeaturesMeta()`. Now the first one doesn't really need to be sorted.

Reply:
Ohhh _that's_ why they were coupled, 👍 I suppose this is happening because we added some CSS features to the readme? I'm confused about `refined-github.css` appearing in the options though 🤔

Reply:
Let's preserve the sorting in both functions to avoid issues

Reply:
> I'm confused about `refined-github.css` appearing in the options though

It's in the readme: https://github.com/refined-github/refined-github/blob/047a6c43b7fae828b95a818194332835bfb71194/readme.md?plain=1#L204